keyboards/gergo/keymaps/germ/keymap.c

```diff
@@ -0,0 +1,198 @@
+ * |--------+------+------+------+------+------|------.           .------|------+------+------+------+------+--------|
+ * |  F1    |  F2  | F3   | F4   | F5   | F6   | BTN1 |           |      | LEFT | DOWN |  UP  | RIGHT|VolDn | VolUp  |
+ * |--------+------+------+------+------+------|------|           |------|------+------+------+------+------+--------|
+ * |  F7    |  F8  | F9   | F10  | F11  | F12  | BTN2 |           |      | MLFT | MDWN | MUP  | MRGHT|Ply/Pa|  Skip  |
+ * `--------+------+------+------+------+-------------'           `-------------+------+------+------+------+--------'
+ *                        .------.   .------.                                 .------.   .-----.
+ *                        |      |   |      |                                 |      |   |     |
+ *                        '------'   '------'                                 `------.   '-----'
+ *                                        ,-------.                     ,-------.
+ *                                        |       |                     | PgUp  |
+ *                                 ,------|-------|                     |-------|------.
+ *                                 |      |       |                     |       |      |
+ *                                 |      |       |                     |       |      |
+ *                                 |      |       |                     |       |      |
+ *                                 `--------------'                     `--------------'
+ */
+[NUMB] = LAYOUT_GERGO(  
+KC_TRNS, KC_1, 	  KC_2,    KC_3,    KC_4,    KC_5,                             KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_TRNS, 
+KC_TRNS, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,         KC_TRNS,  KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, KC_VOLD, KC_VOLU, 
+KC_TRNS, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,        KC_TRNS,  KC_MS_L, KC_MS_D, KC_MS_U, KC_MS_R, KC_MPLY, KC_MNXT, 
+
+                                                  KC_TRNS, KC_TRNS,       KC_PGUP, KC_TRNS,
+                                                           KC_TRNS,       KC_TRNS,
+                                             	  KC_TRNS, KC_TRNS,       KC_TRNS, KC_TRNS)
+};
+
+const uint16_t PROGMEM fn_actions[] = {
+    //[1] = ACTION_LAYER_TAP_TOGGLE(SYMB)                // FN1 - Momentary Layer 1 (Symbols)
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
+{
+  // MACRODOWN only works in this function
+  switch(id) {
+    case 0:
+    if (record->event.pressed) {
+      register_code(KC_RSFT);
+    } else {
+      unregister_code(KC_RSFT);
+    }
+    break;
+  }
+  return MACRO_NONE;
+};
+
+// Runs just one time when the keyboard initializes.
+void matrix_init_user(void) {
+
+};
+
+// Runs constantly in the background, in a loop.
+void matrix_scan_user(void) {
+    //uint8_t layer = biton32(layer_state);
+    biton32(layer_state);
+};
+
+
+/* Keymap template 
```

Comment:
Lines 171-198 should be moved to between the current Lines 137 and 138.

Additionally, if you're intending this to be the default keymap, the folder should be `keyboards/gergo/keymaps/default/` rather than `keyboards/gergo/keymaps/germ/`

If you want to include both a generalized `default` keymap, and a `germ` keymap for your own use, that's fine.